type-reducer/reduce_types.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+rm -rf ../gateway-api/src/apis/processed
+mkdir ../gateway-api/src/apis/processed
+#cargo run -- --apis-dir ../gateway-api/src/apis/standard --out-dir ../gateway-api/src/apis/processed
```

Comment:
```suggestion
```

Reply:
I will remove this file, it is no longer needed.